### ISSUE TITLE
Error in documentation

### ISSUE THREAD

**Issue by Tivin-i** [gravitl/netmaker-docs#94]
### What happened?

Seems there is an error in documentation.
Documentation for install with postgres shows:
`DATABASE = postgres`

When it should be

`DATABASE = postgresql`

### Version

v0.17.0

### Contributing guidelines

- [X] Yes, I did.

**Comment by yearski** [comment]
I believe `DATABASE = postgres` is correct as documented.
[netmaker/database/database.go : getCurrentDB()](https://github.com/gravitl/netmaker/blob/9e3f402e815cab334b27cc59d4a990fe84af1233/database/database.go#L108)

**Comment by Tivin-i** [comment]
you are right, I checked that later and forgot to remove this one.Apologies.Sent from my iPhoneOn 21 Feb 2023, at 00:36, yearski ***@***.***> wrote:﻿
I believe DATABASE = postgres is correct as documented.
netmaker/database/database.go : getCurrentDB()
